big_tests/tests/mod_ping_SUITE.erl

```diff
@@ -30,8 +30,8 @@ all() ->
 
 groups() ->
     G = [{client_ping, [], [ping]},
-         {server_ping, [parallel], all_tests()},
-         {server_ping_kill, [parallel], all_tests()}
+         {server_ping, [], all_tests()},
```

Comment:
Why these tests are not parallel anymore? Is it because of metrics?

Reply:
Yes, I thought it would make sense to check metrics for every test, and as I understand it, they can interfere.

Reply:
I've also tested checking in the `end_per_group` but don't really like the idea, because it may rely on the number of tests in the group, or the length, or the nature of each unit test. In the end, I don't see a way to reliably check the histogram without making assumptions that can backfire...
As for checking metrics after each tests - if we don't do that, then the tests can be parallel as they were, but then the metrics will be untested at all.

Reply:
We could configure a fresh server domain per test case or test group to start separate `mod_ping` instances and have isolated metrics.
It's probably quite a bit of initial setup, but might prove useful when testing other things that could use parallelisation yet require exact metrics.

Reply:
Definitely a good idea but perhaps for a separate refactor PR. :) I think we can live with the lost parallelisation  in this suite.